src/sentry/utils/json.py

```diff
@@ -20,6 +21,7 @@
 from simplejson import JSONDecodeError, JSONEncoder  # noqa: S003
 
 from bitfield.types import BitHandler
+from sentry.features.rollout import in_random_rollout
```

Comment:
ah this is likely to introduce an import (and typing) cycle (since this file gets imported in `settings`) -- probably should be in a separate file for now

Reply:
Could I put the import into the functions that use it?

Reply:
that will eliminate the true-cycle but not the mypy cycle